Source/RageCommunity.Library/Wrappers/NativeGraphicsWrappers.cs

```diff
@@ -59,23 +58,9 @@ public static void DeleteCheckpoint(int handle)
         /// <summary>
         /// When called in a loop, draws a marker with a given marker type with the specified parameters.
         /// </summary>
-        /// <remarks>
-        /// A list of marker types can be found <a href="https://nativedb.dotindustries.dev/natives?search=draw_marker">on NativeDB</a>
```

Comment:
I suggest we keep the link to NativeDB.

Reply:
For what purpose?  The marker types are available in the enum.

Reply:
My bad, you're right, I didn't took a look on the datatype